### PR TITLE
Minor updates to documentation

### LINKED ISSUE #973: typo in error output
If an incorrect option is given in the output file, there's a typo in the warning message: 
image

### LINKED ISSUE #974: Error in hessian options docs
The docs say that `hes_method` can be one of four options:
image
but the code says only `default` and `analytic` are supported:
image

### PR DESCRIPTION
**Pull request by robertapplin**
#### Description of Work
This PR makes some minor updates to the documentation.

Fixes #974
Fixes #973 

#### Testing Instructions

1. Read the docs and make sure they make sense

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?

### CODE REVIEW

File: docs/source/users/options/fitting_option.rst [hunk 1]
```diff
@@ -134,8 +134,6 @@ Current Hessian methods are:
 
 * ``default`` - Hessian information is not passed to minimizers
 * ``analytic`` - uses the analytic Hessian extracted from the fitting problem.
-* ``scipy`` -  uses :ref:`SciPy's finite difference approximations <scipy-hes>`.
-* ``numdifftools`` - uses the python package :ref:`numdifftools <numdifftools-hes>`.
```
Comment (AndrewLister-STFC) [inline docs/source/users/options/fitting_option.rst:138]: I can't recreate the issue. I don't think we should remove this until we can confirm it's not just a test on an outdated version. Can you run it?

**Review by AndrewLister-STFC — CHANGES_REQUESTED** [review]
The f-strings are much neater and fix the typo! But the hessians seems to be a missunderstanding...

File: docs/source/users/options/fitting_option.rst (hunk 1 above)
Comment (robertapplin) [reply docs/source/users/options/fitting_option.rst:138]: Looks like you are right, thanks for finding that 👍 I'll add it back in

Comment (robertapplin) [reply docs/source/users/options/fitting_option.rst:138]: It runs fine for me too

**Review by AndrewLister-STFC — APPROVED** [review]

**Pull request merged**